rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/scan/provider/IndexJavaSourceFilesRuleProvider.java

```diff
@@ -233,7 +233,7 @@ private void addParsedClassToFile(GraphRewrite event, EvaluationContext context,
             {
                 Extendable<?> extendable = (Extendable<?>) javaSource;
                 String superclassName = extendable.getSuperType();
-                if (Strings.isNullOrEmpty(superclassName))
+                if (!Strings.isNullOrEmpty(superclassName))
```

Comment:
Bugs like these will make you pull your hair out sometimes. Fun one to find.

(also, I bet I wrote that line)

Reply:
Yep, written about four years ago. :)

Reply:
youthful mistake :wink: